docs/CHANGELOG.asciidoc

```diff
@@ -89,6 +89,285 @@ This section summarizes the changes in each release.
 
 coming[6.6]
 
+
+[float]
+[[breaking-6.6.0]]
+=== Breaking Changes
+Platform::
+* Upgrades Hapi in legacy platform to v17 {pull}21707[#21707]
+
+[float]
+[[deprecation-6.0.0]]
+=== Deprecations
+Operations::
+* Deprecates tribe settings in 6.x {pull}25548[#25548]
+
+[float]
+[[plugin-6.0.0]]
+=== Plugin API Changes 
+See https://www.elastic.co/blog/kibana-plugin-api-changes-in-6-6[Kibana plugin API changes in 6.6], 
+which describes changes to the visualizations response handler, data schema, and aggregations.
+
+[float]
+[[features-6.0.0]]
+=== New features
+For an overview of the new features that were added in 6.6.0 and how the user experience 
+improved, see the <<release-highlights-6.6.0, 6.6.0 Release Highlights>>.
+
```

Comment:
Let's change this a bit.  Something like:

6.6.0 includes the following new features:

- Index Lifecycle Management
- Remote Clusters and Cross Cluster Replications
- Upgrade Assistant for Elasticsearch 7.0

For more information, see <<release-highlights-6.6.0, 6.6.0 Release Highlights>>.